product-service/src/functions/hello/handler.ts

```diff
@@ -0,0 +1,14 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
```

Comment:
I think you can remove this test lambda and related files. It'll be easier to wok with your codebase.